kotlinx-coroutines-core/jvm/test/linearizability/ChannelsLCStressTest.kt

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2016-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+@file:Suppress("unused")
+
+package kotlinx.coroutines.linearizability
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.channels.*
+import kotlinx.coroutines.channels.TestChannelKind.*
+import kotlinx.coroutines.selects.*
+import org.jetbrains.kotlinx.lincheck.*
+import org.jetbrains.kotlinx.lincheck.annotations.*
+import org.jetbrains.kotlinx.lincheck.annotations.Operation
+import org.jetbrains.kotlinx.lincheck.execution.*
+import org.jetbrains.kotlinx.lincheck.paramgen.*
+import org.jetbrains.kotlinx.lincheck.strategy.stress.*
+import org.jetbrains.kotlinx.lincheck.verifier.*
+import org.junit.*
+import org.junit.runner.*
+import org.junit.runners.*
+import kotlin.reflect.jvm.*
+
+@Param.Params(
+    Param(name = "value", gen = IntGen::class, conf = "1:5"),
+    Param(name = "closeToken", gen = IntGen::class, conf = "1:3")
+)
+class ChannelsLCStressTestImpl: VerifierState() {
+    private val c = channelKind.create()
+
+    @Operation
+    suspend fun send(@Param(name = "value") value: Int) = try {
+        c.send(value)
+    } catch (e : NumberedCancellationException) {
```

Comment:
Style nit: No space before `:` when used to specify variable type (in all the other cases in this file, too)